crates/cli/Cargo.toml

```diff
@@ -40,14 +40,14 @@ assert_cmd = "2"
 diff = "0.1"
 predicates = "3"
 rstest = "0.26"
-wasmparser = "0.214"
-wasmprinter = "0.214"
+wasmparser = "0.238.1"
```

Comment:
We don't really want to bump `wasmparser` & co. until we update Walrus. Otherwise these very heavy dependencies are duplicated.